doc/articles/guides/how-to-create-control-libraries.md

```diff
@@ -57,7 +57,10 @@ You can find the [full sample code](https://github.com/unoplatform/Uno.Samples/b
 
 ## Library assets
 
-WinUI 3 and Uno Platform (4.6 and later) Libraries support the inclusion of content assets to be used with [`StorageFile.GetFileFromApplicationUriAsync`](../features/file-management.md#support-for-storagefilegetfilefromapplicationuriasync), as well as with the `ms-appx://[libraryname]/[assetname_file_name]` format.
+WinUI 3 and Uno Platform (4.6 and later) Libraries support the inclusion of content assets to be used with [`StorageFile.GetFileFromApplicationUriAsync`](../features/file-management.md#support-for-storagefilegetfilefromapplicationuriasync), as well as with the `ms-appx:///[libraryname]/[assetname_file_name]` format.
+
+> [!IMPORTANT]
+> When using library assets with non-WinAppSDK targets, the library name should also be lower cased.
```

Comment:
Really? So referencing something from Uno.Material wouldn't work if you had `"ms-appx:///Uno.Material/blah.blah"`?

Doesn't this new line: `writer.AppendLineInvariantIndented($"private const string __baseUri_prefix_{_fileUniqueId} = \"ms-appx:///{assembly.ToLowerInvariant()}\";");` avoid this issue?

Reply:
There's a bit more than that particular line, but I agree that it's an annoying constraint. It was added like that in 4.6, would make a breaking change for 4.7, but it would not change the fact that case-sensitiveness is required...